src/EcessClient.py

```diff
@@ -31,13 +31,16 @@ async def load(ctx, extension):
     async def unload(ctx, extension):
         client.unload_extension(f'cogs.{extension}')
 
+    # Parent directory of the bot repo; constructed as parentDir(fileDir(file))
+    botDir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Nit: Can you rename this to `bot_dir`? Just to follow the naming conventions everywhere else with snake-case.